### DISCUSSION TITLE
E-Cam V4 Postprocessor

### DISCUSSION THREAD

**Discussion by Team-Deepflights** [terjeio/grblHAL#345] [General]
I own E-Cam V4 to generate the NC files and made a postprocessor for the grbl HAL board, is it possible to add it here for other users?

**Comment by terjeio** [comment]
Not here as this repo is going to be archived soon, I would prefer that you create a repo in your own account that I can link to from the [grblHAL postprocessor repo](https://github.com/grblHAL/PostProcessors).

**Comment by Team-Deepflights** [comment]
OK, I go that way :-)
Tell you after it's fully tested with a real working lathe, for now it's still under construction
